### ISSUE TITLE
Query property id matching fails

### ISSUE THREAD

**Issue by l-hoang** [KatanaGraph/katana#449]
```
 ./graph-query ldbc_003/meta --query "match (a:Person {id:21}) return a;"
....
ERROR: /home/loc/katana/libquery/src/Table.cpp:279: No column found with alias 'a'
```

I can confirm a person with id 21 does exist in the ldbc 003 graph.

Trying this also fails with a different error:

```
 ./graph-query ldbc_003/meta --query "match (a:Person) where a.id=21 return a;"
...
ERROR: /home/loc/katana/libquery/src/FilterOp.cpp:30: Cannot find property in graph
```

@roshandathathri @bowu any thoughts? does the id property simply not exist on Person nodes?

**Comment by l-hoang** [comment]
note that this is a priority for the queries for the august deadline as they match via id

**Comment by bowu** [comment]
Are you sure that AttributedGraph has a vertex property called "id"?

**Comment by l-hoang** [comment]
> Are you sure that AttributedGraph has a vertex property called "id"?

it does; see the table I posted in slack yesterday

**Comment by bowu** [comment]
This issue has been resolved by PR #453
